stdlib/websub/src/main/ballerina/src/websub/subscriber_service_endpoint.bal

```diff
@@ -242,11 +279,11 @@ public type ExtensionConfig record {|
     map<map<map<[string, typedesc<record {}>]>>>? headerAndPayloadKeyResourceMap = ();
 |};
 
-# The function called to discover hub and topic URLs defined by a resource URL.
+# Discovers hub and topic URLs defined by a resource URL.
 #
 # + resourceUrl - The resource URL advertising hub and topic URLs
 # + publisherClientConfig - The configuration for the publisher client
-# + return - `(string, string)` (hub, topic) URLs if successful, `error` if not
+# + return - A `(hub, topic)` as a `(string, string)` if successful or else an `error` if not
```

Comment:
```suggestion
# + return - A `(hub, topic)` as a `(string, string)` if successful or else an `error` if not
```
Need to specify the error.

Reply:
We are using a common error in this module. So we don't need to specify the error.